src/components/Person/Person.jsx

```diff
@@ -1 +1,42 @@
-// export const Person = ({ person }) => ();
+export const Person = ({ person }) => (
+  <section className="Person">
+    <h2 className="Person__name">
+      {`My name is ${person.name}`}
```

Comment:
you can use without a template string
```suggestion
      My name is {name}
```